subsys/net/ip/tcp2.c

```diff
@@ -58,10 +58,29 @@ static size_t tcp_endpoint_len(sa_family_t af)
 		sizeof(struct sockaddr_in6);
 }
 
+static union tcp_endpoint *tcp_endpoint_alloc(void)
+{
+	union tcp_endpoint *ep;
+
+	ep = tcp_calloc(1, NET_SOCKADDR_MAX_SIZE);
+	if (!ep) {
+		return NULL;
+	}
+
+	memset(ep, 0, NET_SOCKADDR_MAX_SIZE);
```

Comment:
The tcp_calloc() will call k_calloc() which zeroes the area, so this memset can be removed.

Reply:
done.